src/app/asset-grid/asset-grid.component.ts

```diff
@@ -653,6 +654,11 @@ export class AssetGrid implements OnInit, OnDestroy {
           this.isLoading = false;
           this.allResults = allThumbnails
           this.results = this.allResults.slice(0)
+          this.visibleResults = []
+          for (let i = 0; i < this.results.length; i++) {
+              if (this.results[i].status !== 'not-available')
+                this.visibleResults.push(this.results[i])
```

Comment:
nit - I think this this could be simplified slightly by using a filter on the results array. 

 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
Done!